spyder/utils/stylesheet.py

```diff
@@ -291,42 +293,49 @@ def set_stylesheet(self):
             marginTop=self.TOP_MARGIN,
             paddingTop='4px',
             paddingBottom='4px',
-            paddingLeft='4px' if is_macos else '10px',
-            paddingRight='10px' if is_macos else '4px'
+            paddingLeft='4px' if MAC else '10px',
+            paddingRight='10px' if MAC else '4px'
         )
 
-        # Show tabs left-aligned on Mac
         if MAC:
+            # Show tabs left-aligned on Mac and remove spurious
+            # pixel to the left.
             css.QTabBar.setValues(
-                alignment='left'
+                alignment='left',
+                marginLeft='-1px'
             )
 
             css['QTabWidget::tab-bar'].setValues(
-                alignment='left'
+                alignment='center',
+            )
+        else:
+            # Remove spurious pixel to the left
+            css.QTabBar.setValues(
+                marginLeft='-1px'
```

Comment:
@dalthviz, please change this value to `-2px`, `-3px` or less until you find one that fixes the glitch for you. That should work. If not, let me know because I'll have to resort to the previous approach but increasing a different margin.

Reply:
Using `-3px` I'm unable to see the whit dot :+1: :

![imagen](https://user-images.githubusercontent.com/16781833/204620837-c4bdf628-4c08-4f50-b77e-ec88936b622b.png)

Reply:
Great! I forced pushed my last commit with the new value for Windows.